CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+# 1.4.1
+- Updating mock server with support for PDF type
+- Updating test-tool with testing validity of tickets for PDF type
```

Comment:
This is a changelog for the specification so no changes here.

Reply:
revoked